internal/shared/clients.go

```diff
@@ -44,16 +44,16 @@ import (
 
 // ClientFactory are shared clients and configurations for use across the CLI commands (cmd) and handlers (pkg).
 type ClientFactory struct {
-	API           func() api.APIInterface
-	AppClient     func() *app.Client
-	AuthInterface func() auth.AuthInterface
-	CLIVersion    string
-	Config        *config.Config
-	EventTracker  tracking.TrackingManager
-	HookExecutor  hooks.HookExecutor
-	IO            iostreams.IOStreamer
-	Runtime       runtime.Runtime
-	SDKConfig     hooks.SDKCLIConfig
+	API          func() api.APIInterface
+	AppClient    func() *app.Client
+	Auth         func() auth.AuthInterface
```

Comment:
note: This is the main change

Reply:
I like that this also removes a space in the `struct`! 👾 ✨